specs/pvs/corla.pvs

```diff
@@ -981,6 +999,8 @@ BEGIN
   IMPORTING elections,
             FAFSSL
 
+%%% Audit board members are members of the public who come from different political parties. According to 
+
```

Comment:
So far, nowhere. It would be enforced by an invariant check on the audit board selection endpoint, but right now, political party is just a String... so if one is "REPUBLICAN" and the other is "Republican!", that'd be considered different. We may need a table of political parties to select from (for phase 2 or 3, not phase 1).